crates/biome_js_parser/tests/js_test_suite/error/import_source_err.js.snap

```diff
@@ -0,0 +1,151 @@
+                        name_token: IDENT@37..44 "source" [] [Whitespace(" ")],
+                    },
+                },
+                comma_token: missing (required),
+                specifier: JsNamespaceImportSpecifier {
+                    star_token: STAR@44..46 "*" [] [Whitespace(" ")],
+                    as_token: AS_KW@46..49 "as" [] [Whitespace(" ")],
+                    local_name: JsIdentifierBinding {
+                        name_token: IDENT@49..51 "x" [] [Whitespace(" ")],
+                    },
+                },
+                from_token: FROM_KW@51..56 "from" [] [Whitespace(" ")],
+                source: JsModuleSource {
+                    value_token: JS_STRING_LITERAL@56..59 "\"x\"" [] [],
+                },
+                assertion: missing (optional),
+            },
+            semicolon_token: SEMICOLON@59..60 ";" [] [],
+        },
+    ],
+    eof_token: EOF@60..60 "" [] [],
+}
+```
+
+## CST
+
+```
+0: JS_MODULE@0..60
+  0: (empty)
+  1: (empty)
+  2: JS_DIRECTIVE_LIST@0..0
+  3: JS_MODULE_ITEM_LIST@0..60
+    0: JS_IMPORT@0..29
+      0: IMPORT_KW@0..7 "import" [] [Whitespace(" ")]
+      1: JS_IMPORT_COMBINED_CLAUSE@7..28
+        0: JS_DEFAULT_IMPORT_SPECIFIER@7..14
+          0: JS_IDENTIFIER_BINDING@7..14
+            0: IDENT@7..14 "source" [] [Whitespace(" ")]
+        1: (empty)
+        2: JS_NAMED_IMPORT_SPECIFIERS@14..20
+          0: L_CURLY@14..16 "{" [] [Whitespace(" ")]
+          1: JS_NAMED_IMPORT_SPECIFIER_LIST@16..18
+            0: JS_SHORTHAND_NAMED_IMPORT_SPECIFIER@16..18
+              0: (empty)
+              1: JS_IDENTIFIER_BINDING@16..18
+                0: IDENT@16..18 "x" [] [Whitespace(" ")]
+          2: R_CURLY@18..20 "}" [] [Whitespace(" ")]
+        3: FROM_KW@20..25 "from" [] [Whitespace(" ")]
+        4: JS_MODULE_SOURCE@25..28
+          0: JS_STRING_LITERAL@25..28 "\"x\"" [] []
+        5: (empty)
+      2: SEMICOLON@28..29 ";" [] []
+    1: JS_IMPORT@29..60
+      0: IMPORT_KW@29..37 "import" [Newline("\n")] [Whitespace(" ")]
+      1: JS_IMPORT_COMBINED_CLAUSE@37..59
+        0: JS_DEFAULT_IMPORT_SPECIFIER@37..44
+          0: JS_IDENTIFIER_BINDING@37..44
+            0: IDENT@37..44 "source" [] [Whitespace(" ")]
+        1: (empty)
+        2: JS_NAMESPACE_IMPORT_SPECIFIER@44..51
+          0: STAR@44..46 "*" [] [Whitespace(" ")]
+          1: AS_KW@46..49 "as" [] [Whitespace(" ")]
+          2: JS_IDENTIFIER_BINDING@49..51
+            0: IDENT@49..51 "x" [] [Whitespace(" ")]
+        3: FROM_KW@51..56 "from" [] [Whitespace(" ")]
+        4: JS_MODULE_SOURCE@56..59
+          0: JS_STRING_LITERAL@56..59 "\"x\"" [] []
+        5: (empty)
+      2: SEMICOLON@59..60 ";" [] []
+  4: EOF@60..60 "" [] []
+
+```
+
+## Diagnostics
+
+```
+import_source_err.js:1:15 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × expected `,` but instead found `{`
+  
+  > 1 │ import source { x } from "x";
```

Comment:
I don't think the message is correct here

Reply:
i will adjust the syntax error here to align with babel

Reply:
done~